app/src/test/java/com/x8bit/bitwarden/ui/vault/feature/item/VaultItemScreenTest.kt

```diff
@@ -906,9 +935,9 @@ class VaultItemScreenTest : BaseComposeTest() {
 
         composeTestRule.assertNoDialogExists()
 
-        composeTestRule
-            .onNodeWithText("Restore")
-            .performClick()
+        mutableStateFlow.update {
+            it.copy(dialog = VaultItemState.DialogState.RestoreItemDialog)
```

Comment:
So this is similar to the previous test.

We just need a test to verify that clicking the button sends the appropriate event.

Reply:
Hmmm, I may have misread this, is this just a copy of the new test you added above?

It looks like it just verifies the dialog shows and hides based on state, right?